### DISCUSSION TITLE
Gradience archived

### DISCUSSION THREAD

**Discussion by UcraNerd** [end-4/dots-hyprland#667] [General]
Gradience project got archived 2 days ago and they are looking for new maintainer(s). I think that in the future or even in a few months, if the project remains like this, the app could no longer work or have some new bugs due to the adwaita theme updates. I wonder what you guys think about this situation.

**Comment by end-4** [comment]
i'll introduce an "adwaita blue" mode that just makes gtk not use gradience customization, just in case theming bugs happen. with this mode you'll just get the normal blue adwaita and you'll get proper light/dark switching for gtk4 apps
